src/views/docs/Docs.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-<div id="docs-wrapper">
+<div id="docs-wrapper" :class="this.$vuetify.theme.isDark ? 'docs-wrapper_dark' : 'docs-wrapper_light'">
```

Comment:
'docs-wrapper_light' is absent, but I think this condition is not needed anymore

Reply:
@m-kus thanks, cleared

Reply:
@m-kus the condition is needed to use its result in the 226th line